stale.sh

```diff
@@ -139,6 +144,19 @@ list_keypair() {
 	done
 }
 
+list_image() {
+	for resource_id in $(openstack image list -f value -c ID); do
+		res="$(openstack image show -f json -c updated_at -c name "$resource_id")"
+		update_time="$(jq -r '.updated_at' <<< "$res")"
+		name="$(jq -r '.name' <<< "$res")"
+		if [[ "$name" =~ rhcos-.{8}-.{5} ]] || [[ "$name" =~ .{8}-.{5}-.{5}-rhcos ]] || [[ "$name" =~ bootstrap-ign-.{8}-.{5}-.{5} ]]; then
```

Comment:
:open_mouth:

Reply:
oooh OK the names the Installer picks for images, based on the cluster id

Reply:
If you want to be kind to the next generation, you might add a comment. this is the RHCOS image for IPI, this is for UPI, this is the bootstrap ignition payload...

Reply:
Thanks, I added some comments as you suggested and realized I was missing the ignition for IPI.